pkg/syncer/types.go

```diff
@@ -20,19 +20,71 @@ import (
 	"sync"
 
 	corelisters "k8s.io/client-go/listers/core/v1"
+	"k8s.io/api/core/v1"
 
 	cnsvsphere "sigs.k8s.io/vsphere-csi-driver/pkg/common/cns-lib/vsphere"
 	cnsconfig "sigs.k8s.io/vsphere-csi-driver/pkg/common/config"
 	k8s "sigs.k8s.io/vsphere-csi-driver/pkg/kubernetes"
 )
 
+const (
+	// default interval for csi full sync, used unless overridden by user in csi-controller YAML
+	defaultFullSyncIntervalInMin = 30
+
+	// Constants for specifying operation that needs to be performed on CNS volume
+	// Create the volume on CNS
+	createVolumeOperation = "createVolume"
+	// Update the volume entries on CNS
+	updateVolumeOperation = "updateVolume"
+	// Delete the PVC entry and Pod entry (if it exists) on CNS
+	updateVolumeWithDeleteClaimOperation = "updateVolumeWithDeleteClaim"
+	// Delete the Pod entry on CNS
+	updateVolumeWithDeletePodOperation = "updateVolumeWithDeletePod"
+
+	// Prefix of log message for fullSync interval set in env variable
+	fullSyncIntervalLogMessagePrefix = "FullSync: fullSync interval set in env variable FULL_SYNC_INTERVAL_MINUTES"
```

Comment:
Let's define `FULL_SYNC_INTERVAL_MINUTES` as const. also the prefix message in the const does not look good. value is more than just a prefix.